graylog2-server/src/main/java/org/graylog2/contentpacks/EntityDescriptorIds.java

```diff
@@ -58,7 +58,7 @@ public static EntityDescriptorIds of(Collection<EntityDescriptor> entityDescript
     }
 
     public static boolean isDefaultStreamDescriptor(EntityDescriptor descriptor) {
-        return ModelTypes.STREAM_V1.equals(descriptor.type()) && Stream.isDefaultStreamId(descriptor.id().id());
+        return ModelTypes.STREAM_V1.equals(descriptor.type()) && Stream.isSystemStreamId(descriptor.id().id());
```

Comment:
Shouldn't we rename the method to `isSystemStreamDescriptor`?

Reply:
makes sense :+1: